### DISCUSSION TITLE
VSCode not recognizing parameter names on MapStruct mappings

### DISCUSSION THREAD

**Discussion by kjq** [mapstruct/mapstruct#3672] [Q&A] [answered]
Anyone see this error in VSCode when autobuilding tests?  

It appears that VSCode, or something, cannot recognize the parameters specifically in MapStruct mappings.  

This works fine in the command-line and if I disable autobuild during tests and build the code first outside of VSCode the test will run fine.

https://github.com/microsoft/vscode-java-debug/issues/1314

**Comment by filiphr** [comment, marked as answer]
I'm not a VSCode user. However, to me it looks like that the VSCode extension does not honor `<maven.compiler.parameters>true</maven.compiler.parameters>` and then the auto build (which I guess does some build compiled files) stops working. That's why you are seeing those `arg0, arg1`.

**Reply by kjq** [reply]
Yeah, it _used_ to work in VSCode fine.
